_episodes/13-scheduler.md

```diff
@@ -91,15 +92,10 @@ the *queue*. To check on our job's status, we check the queue using the command
 ```
 {{ site.host_prompt }} {{ site.sched_status }} {{ site.sched_flag_user }}
 ```
-{: .bash}
```

Comment:
This `{:bash}` tag belongs to the preceding block: its deletion here is in error. Please restore.